cli/internal/server/turbo.proto

```diff
@@ -0,0 +1,53 @@
+syntax = "proto3";
+
+option go_package = "github.com/vercel/turborepo/cli/internal/server";
+
+package server;
+
+service Turbo {
```

Comment:
Should we call this `Turborepo`?

Reply:
I don't have a strong opinion on this, except to point out that I don't intend for this to a public interface. If/when we add any sort of plugin functionality, I would expect it to be a separate service definition that we take a lot more care around version compatibility and stability. 

As it stands, the current protocol shuts the daemon down on anything other than exact version match, so the scope of compatibility we need is very narrow, and that will give us a lot of freedom to tweak this definition. However, this specific piece is within that scope of compatibility. So we should pick something we can live with internally and won't regret not having available for public consumption later.

Reply:
Renamed to `Turbod` to distinguish our internal protocol